graylog2-web-interface/src/components/sidecars/configuration-forms/ConfigurationForm.jsx

```diff
@@ -147,6 +148,14 @@ const ConfigurationForm = createReactClass({
     this.setState({ formData: nextFormData });
   },
 
+  _onTemplateImport(nextTemplate) {
+    const nextFormData = lodash.cloneDeep(this.state.formData);
+    if (!nextFormData.template || window.confirm('Do you want to overwrite your current work with this Configuration?')) {
+      nextFormData.template = nextTemplate;
+      this.setState({ formData: nextFormData });
```

Comment:
This does not reload the template validations, so we should replace this two lines with:

```js
this._onTemplateChange(nextTemplate);
```

To reproduce the issue, use an empty source code editor and then import a configuration. The field will still be red because it thinks it is empty.

The same issue happens when adding the default collector configuration, so you could also change it there if you want to 🙂